GETTING-STARTED.md

```diff
@@ -5,14 +5,14 @@ docs. If you're here, you're likely looking for guidance on how to write
 documentation for the project. You're in luck! This is where we house all of
 that information.
 
-This guide is divided into **why** we're writing docs, **what** we want to
-write, and **how** we write it.
+This guide is divided into sections for **why** we're writing docs, **what** we
+want to write, and **how** we write it.
 
-* If you're new here, start from the top and go down.
-* If you're ready to write some docs, but don't know what to write, look at the
-second section.
-* If you're having trouble with formatting, or have a grammar question, look at
-the last section.
+* If you're new here, start with [Why We Write Docs section][].
```

Comment:
I kind of miss the "and go down" from the original text — would you be greatly opposed to reintroducing that?